search-relevancy-test/README.md

```diff
@@ -55,13 +55,19 @@ Arguments:
 
 `lein run analyze-test <anomaly-number> <anomaly-filename>`
```

Comment:
This task doesn't work for the top-n-tests case. We could add another analyze-top-n-test to compare the collection in the top-n file and the actual collection that's in the top-n position.  This could be added later. Just need to modify the doc to say so.

Reply:
I clarified the two types of tests in the README file and updated the analyze-test documentation to clarify it only supports X before Y tests.